src/features/terms-and-privacy/hooks.ts

```diff
@@ -0,0 +1,48 @@
+import { useCallback, useEffect, useState } from 'react';
+import {
+  getAcceptedLegalDocsVersion,
+  setAcceptedLegalDocsVersion,
+} from './services';
+import { CURRENT_VERSION } from './config';
+
+export const useAcceptDocs = () => {
+  const [accept, setAccept] = useState(false);
```

Comment:
Can we use `isAccepted` or `accepted` instead of `accept`? To me the meaning of `accept` and `setAccept` is the same, or pretty similar.

In file **src/ui/app/pages/welcome.jsx** 

```
const { accept, setAccept } = useAcceptDocs();
<Button isDisabled={!select || !accept}
...
```
I would use `isDisabled={!select || !isAccepted}`

Reply:
btw. I would also rename `select` in this part of code: `const [select, setSelect] = React.useState(null);` to `isSelected` or `selected`